Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -187,18 +187,7 @@ void Renderer::ReinterpretPixelData(EFBReinterpretType convtype)
 
 u16 Renderer::BBoxRead(int index)
```

Comment:
Could these wrapper/impl functions be removed now?

Shader looks okay to me.

Reply:
I was thinking we'd just keep that around as a stepping stone for abstracting bbox more later.